### PR TITLE
Fix kicker on Safari

### PR DESCRIPTION
**Pull request by regiskuckaertz**
The kicker had a 3px border all around, this fix removes it. Also slightly tweak the border height to go a couple of pixels below the baseline.

### CODE REVIEW

File: static/src/stylesheets/module/facia-garnett/_item.scss [hunk 1]
```diff
@@ -197,7 +197,7 @@ $fc-item-gutter: $gs-gutter / 4;
     margin-right: $gs-gutter / 4;
     font-weight: 700;
     border-right-style: solid;
-    border-right-width: 1px;
+    border-width: 0 1px 0 0;
```
Comment (sndrs) [inline static/src/stylesheets/module/facia-garnett/_item.scss]: where was the 3px coming from?

Comment (regiskuckaertz) [reply static/src/stylesheets/module/facia-garnett/_item.scss]: It looks like a Safari-only issue: https://jsfiddle.net/8t0n2u6s/

Comment (regiskuckaertz) [reply static/src/stylesheets/module/facia-garnett/_item.scss]: also seeing it in FF 😂 that's weird.

Comment (sndrs) [reply static/src/stylesheets/module/facia-garnett/_item.scss]: ooooh, turns out [initial value of `border-width` is `medium`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-width)! who knew?

Comment (sndrs) [reply static/src/stylesheets/module/facia-garnett/_item.scss]: maybe just `border-right: 1px solid transparent` then?

**Review by GHaberis — APPROVED** [review]
great 👍

**Comment by blongden73** [comment]
@regiskuckaertz Nice one 👍

**Review by blongden73 — APPROVED** [review]

**Pull request merged**